app/languages/en/static/welcomeScreen.json

```diff
@@ -1,23 +1,13 @@
 {
-  "welcome_to_ttc": "Welcome to Plant-for-the-Planet App",
-  "welcome_screen_summary": "Create your Personal Tree Counter.",
-  "welcome_screen_summary_1": "Support Tree Planting Organizations from all over the world and start planting trees from your phone!",
-  "view_global_treecounter": " View Global Tree Counter",
-  "welcome_scrn_1_heading": "You’re probably here\nto plant some trees",
-  "welcome_scrn_1_sub_heading": "but that’s just the start…",
-  "welcome_scrn_1_get_started_btn": "Get Started",
-  "open_as_guest": "Continue as Guest",
-  "welcome_scrn_2_heading": "Tree Donations\nSimplified",
-  "welcome_scrn_2_sub_heading": "Find large scale reforestation\nprojects and help plant more\nwith few taps.",
-  "welcome_scrn_3_heading": "A Tree Counter\nfor your trees",
-  "welcome_scrn_3_sub_heading": "Have you ever thought\nof starting your own forest\nto save some trees?",
-  "welcome_scrn_4_heading": "Commission Free\n& Open Source",
-  "welcome_scrn_4_sub_heading": "We’re serious!\n100% donations go directly to\ntree planting organizations",
-  "welcome_scrn_4_commision": "COMMISSION",
-  "welcome_scrn_5_heading": "Keep track of\ntrees you plant ",
-  "welcome_scrn_5_sub_heading": "Registering a tree you’ve\nplanted is a breeze and you can\nalways add details about it.",
-  "create_an_account": "Create an Account",
-  "login": "Log in",
-  "already_an_account": "Already have an account?",
-  "continue": "Continue"
+  "login": "Log in / Sign Up",
+  "continue": "Continue",
+  "tree_mapper": "Tree Mapper",
+  "tree_inventory": "Tree Inventory",
+  "tree_inventory_sub_header": "of draft and pending registrations",
```

Comment:
This should be separated by span/div etc, and converted into a single string 
`Tree Inventory <span> of draft and pending registrations </span>`
same applies for Download Maps for offline use